sdk/translation/ai-translation-text-rest/tsp-location.yaml

```diff
@@ -0,0 +1,3 @@
+directory: specification/translation/Azure.AI.TextTranslation
+commit: 169a28b2d5f04417a19c0a877af2a7a9ad195fb2
+repo: MikeyMCZ/azure-rest-api-specs
```

Comment:
I think we have the rule for release that it could only be generated from the specs in the main repo instead of the personal fork.  So could you update the location of the api spec?

Reply:
Oh, I forgot to switch to the correct location. Fixed.